node/src/chain_spec.rs

```diff
@@ -189,5 +189,10 @@ fn testnet_genesis(
 				})
 				.collect(),
 		}),
+		pallet_faucet: Some(FaucetConfig {
+			rampage: true,
```

Comment:
rampage mint always allowed?

Reply:
i disabled in parachain spec

Reply:
Yes the idea was to disable it in parachain. We'll probably need different chainspec build functions for multi runtime env anyway as we don't want to pre-mint exogenous tokens on mainnet/roccocco (maybe different ones) and we want to do that on testnets.